typings/index.d.ts

```diff
@@ -3095,6 +3097,7 @@ export const Constants: {
         size: AllowedImageSize,
         dynamic: boolean,
       ): string;
+      AvatarDecoration: (userId: Snowflake, hash: string, format: AllowedImageFormat, size: AllowedImageSize) => string;
```

Comment:
```suggestion
      AvatarDecoration(userId: Snowflake, hash: string, format: AllowedImageFormat, size: AllowedImageSize): string;
```
You could make it fit with the rest of the endpoints here